src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/WorkspaceProjectContextProvider.cs

```diff
@@ -86,6 +86,8 @@ public async Task ReleaseProjectContextAsync(IWorkspaceProjectContextAccessor ac
                                                                                     hostObject,
                                                                                     data.BinOutputPath);
 
+                context.SetProperty(ConfigurationGeneral.TargetFrameworkProperty, data.TargetFramework ?? "");
```

Comment:
I know we're going to back this out in 16.10 but it would be useful to have a comment explaining why we need to send the `TargetFramework`.